src/overlays/actors/ovl_En_Sekihi/z_en_sekihi.c

```diff
@@ -32,20 +36,147 @@ const ActorInit En_Sekihi_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+static s16 sObjectIds[] = { OBJECT_SEKIHIL, OBJECT_SEKIHIG, OBJECT_SEKIHIN, OBJECT_SEKIHIZ, OBJECT_ZOG };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Sekihi/EnSekihi_Init.s")
+static s32 sOpaDLists[] = { &gGraveTriforceSymbolDL, &gGraveGoronSymbolDL, &gGraveKokiriSymbolDL, &gTombSongOfSoaringDL,
+                            &gMikauGraveDL };
```

Comment:
```suggestion
static Gfx* sOpaDLists[] = { gGraveTriforceSymbolDL, gGraveGoronSymbolDL, gGraveKokiriSymbolDL, gTombSongOfSoaringDL,
                            gMikauGraveDL, };
```
These are DLs so proper type is `Gfx*`, also the `&` are not needed as `DL`s are arrays.
Note, this applies to `sXluDLists` as well, and will need to change the variable types in the struct as well.
Note2, since the names a pretty long a trailing comma will probably format better.